experimental/host/mt7688/gpio.go

```diff
@@ -67,54 +67,104 @@ var (
 // the GPIO Pin Function Mapping on page 108:
 // https://labs.mediatek.com/fileMedia/download/9ef51e98-49b1-489a-b27e-391bac9f7bf3
-var mappingMT7688 = map[string][3]pin.Func{
-	"I2S_SDI":                      {"I2SSDI", "PCMDRX"}, // 0
-	"I2S_SDO":                      {"I2SSDO", "PCMDTX"},
-	"I2S_WS":                       {"I2SW", "PCMCLK"},
-	"I2S_CLK":                      {"I2SCLK", "PCMFS"},
-	"I2C_SCLK":                     {"I2C_SCLK"},
-	"I2C_SD":                       {"I2C_SD"}, // 5
-	"SPI_CS1":                      {"SPI_CS1", "REF_CLKO"},
-	"SPI_CLK":                      {"SPI_CLK"},
-	"SPI_MOSI":                     {"SPI_MOSI"},
-	"SPI_MISO":                     {"SPI_MISO"},
-	"SPI_CS0":                      {"SPI_CS0"}, // 10
-	"GPIO0":                        {"GPIO#11", "REF_CLKO", "PERST_N"},
-	"UART_TXD0":                    {"UART_TXD0"},
-	"UART_RXD0":                    {"UART_RXD0"},
-	"MDI_TP_P1":                    {"SPIS_CS", "", "PWM_CH0"},
-	"MDI_TN_P1":                    {"SPIS_CLK", "", "PWM_CH1"}, // 15
-	"MDI_RP_P1":                    {"SPIS_MISO", "", "UART_TXD2"},
-	"MDI_RN_P1":                    {"SPIS_MOSI", "", "UART_RXD2"},
-	"MDI_RP_P2":                    {"PWM_CH0", "", "eMMC_D7"},
-	"MDI_RN_P2":                    {"PWM_CH1", "", "eMMC_D6"},
-	"MDI_TP_P2":                    {"UART_TXD2", "PWM_CH2", "eMMC_D5"}, // 20
-	"MDI_TN_P2":                    {"UART_RXD2", "PWM_CH3", "eMMC_D4"},
-	"MDI_TP_P3":                    {"SD_WP"}, // todo: add aliases mapping eMMC
-	"MDI_TN_P3":                    {"SD_CD"},
-	"MDI_RP_P3":                    {"SD_D1"},
-	"MDI_RN_P3":                    {"SD_D0"}, // 25
-	"MDI_RP_P4":                    {"SD_CLK"},
-	"MDI_RN_P4":                    {"SD_CMD"},
-	"MDI_TP_P4":                    {"SD_D3"},
-	"MDI_TN_P4":                    {"SD_D2"},
-	"EPHY_LED4_N_JTRST_N (7688KN)": {"EPHY_LED4_K", "", "JTAG_RST_N"}, // 30
-	"EPHY_LED3_N_JTCLK (7688KN)":   {"EPHY_LED3_K", "", "JTAG_CLK"},
-	"EPHY_LED2_N_JTMS (7688KN)":    {"EPHY_LED2_K", "", "JTAG_TMS"},
-	"EPHY_LED1_N_JTDI (7688KN)":    {"EPHY_LED1_K", "", "JTAG_TDI"},
-	"EPHY_LED0_N_JTDO (7688KN)":    {"EPHY_LED0_K", "", "JTAG_TDO"},
-	"WLED_N (7688KN)":              {"WLED_N"}, // 35
-	"PERST_N":                      {"PERST_N"},
-	"REF_CLKO":                     {"REF_CLKO"},
-	"WDT_RST_N":                    {"WDT_RST_N"},
-	"EPHY_LED4_N_JTRST_N (7688AN)": {"EPHY_LED4_N", "", "JTAG_RST_N"},
-	"EPHY_LED3_N_JTCLK (7688AN)":   {"EPHY_LED3_N", "", "JTAG_CLK"}, // 40
-	"EPHY_LED2_N_JTMS (7688AN)":    {"EPHY_LED2_N", "", "JTAG_TMS"},
-	"EPHY_LED1_N_JTDI (7688AN)":    {"EPHY_LED1_N", "", "JTAG_TDI"},
-	"EPHY_LED0_N_JTDO (7688AN)":    {"EPHY_LED0_N", "", "JTAG_TDO"},
-	"WLED_N (7688AN)":              {"WLED_N"},
-	"UART_TXD1":                    {"UART_TXD1", "PWM_CH0"}, // 45
-	"UART_RXD1":                    {"UART_RXD1", "PWM_CH1"},
+var mapping = [][3]pin.Func{
+	{"I2SSDI", "PCMDRX"}, // 0
```

Comment:
In the next commit, you can update the function names.

Reply:
Resolved via #313